Yoast/javascript
# Add function words assessment for Farsi

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Adds the function words assessment for Farsi and edits a typo in the Arabic function words file.
* [Yoast SEO Free] Improves keyphrase recognition in Farsi by filtering the function words such as `هفت`, `چهارم`, `یا `.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

Test the following in WP SEO Free:

- Set the site language to Farsi
- Create a new post and add a text with at least 150 words
- Set your keyphrase to `نوزده گربه`
- Add the word `گربه` to the text
- Make sure that the keyphrase density shows that the keyphrase is found once

Extra team Lingo test instructions: run `yarn test` and make sure all tests pass
## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes https://yoast.atlassian.net/browse/LIN-542
